config/routes.rb

```diff
@@ -2,5 +2,11 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  # root "users#index"
+  
+  get '/', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
+  
```

Comment:
- [ ] [OPTIONAL] Kindly write the routes this way
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
 end
```
and make the `users#index` action the default route by uncommenting the `# root "users#index"` line as it's more professional and easier to read and write.

Reply:
Alright, thank you :smiley: